registry/nacos/registry_test.go

```diff
@@ -35,6 +36,9 @@ import (
 )
 
 func TestNacosRegistry_Register(t *testing.T) {
+	if _, err := http.Get("http://console.nacos.io/nacos/"); err != nil {
```

Comment:
maybe u'd better encapsulate a func to delete duplicate codes.

Reply:
done